content/en/continuous_testing/explorer/saved_views.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Saved Views
+kind: documentation
+further_reading:
+- link: "/continuous_testing/explorer/search_syntax/"
+  tag: "Documentation"
+  text: "Learn how to create a search query"
+---
+
+## Overview
+
+Saved views allow you to save the state of the [Synthetic Monitoring & Continuous Testing Explorer][2] and enable effective troubleshooting by providing you with quick access to scoped queries, relevant facets, visualization options, and the time range. 
+
+Saved views can keep track of your:
+
+- CI batches and test runs
+- Search queries (such as failed test runs with HTTP error status codes, failing test runs in the CI by their blocking status, test runs that required retries, and test IDs to add to your CI pipeline)
+- Column sort order
+- Live time range (such as the past hour or the past week)
+- Visualizations (such as a timeseries, toplist, table, or funnel graph)
+- Subset of facets
```

Comment:
facets are filters that are used in the search query, so we can get rid from them here as we talk about search above

Reply:
```suggestion
```